apps/wing/src/analytics/collect.ts

```diff
@@ -18,11 +18,16 @@ export async function collectCommandAnalytics(cmd: Command): Promise<string | un
   const osCollector = new OSCollector();
   const nodeCollector = new NodeCollector();
   const ciCollector = new CICollector();
-  const gitCollector = new GitCollector();
   const cliCollector = new CLICollector(cmd);
 
+  // If entrypoint to app is provided, we will give that to git collector to use for
+  // running queries against the git repo, otherwise we will use the current working directory
+  const gitCollector = new GitCollector({
+    appEntrypoint: cmd.args.length > 0 ? cmd.args[0] : '.'
```

Comment:
Is there any concern that this data can possibly contain PII? e.g. an absolute path could have a name.

Reply:
Name is considered PII, but not when it is the only thing that we collect 

> That means that something like a first name is not PII, because it’s likely that many people have that first name. However, a full name — first, middle, and last — is considered PII by many organizations. It’s much less likely that many people have that same full name.

We can easily use `path.basename` 

@hasanaburayyan has asked me about this before, and I didn't consider the PII aspect

Reply:
So Im curious @ekeren if someone for some strange reason had their absolute path as
<first_name>/<last_name>/<social_security_number>/<address>/repos/my-wing-app

we would technically be collecting PII right?

I mean I guess the same could be said if the entrypoint file was `hasan_aburayyan.w`

Reply:
I think we should just avoid automatically collecting user-entered data whenever possible, hashing it for tracking whenever needed

Reply:
From the point of avoiding collecting user-entered data then that would mean we also should skip collecting what flags and parameters were passed. IE `wing compile -t tf-aws app.w --plugin some_path/plugin.js` could also result in pii. 

I mean we could collect what --options were used but ignore any values passed to them. It would still give a sense for usage I think

Reply:
For options with known values (like -t) we can collect those known values, skipping unexpected ones that may have been accidentally entered

Reply:
We are going to deep here, we aren't collecting name deliberately. it might be part of a file, part of a path, part of plugin, etc...

If I go to any website and add a `?utm_campaign="eyal keren, <id>"` it doesn't mean that they are collecting PII 

No need to change the code, it is fine to collect the full path.

Reply:
Is accidental PII not still PII? If we can avoid it then I think we should.

Reply:
This should be a team decision, and I want to make sure we all get a chance to voice our opinions.
Let's remove that potentially-PII field from our analytics for now and discuss it tomorrow on team time. If the team decides, we can add it back in tomorrow.

@hasanaburayyan - please let's get that merged in after this minor change.